modules/sandbox/src/Sandbox.scala

```diff
@@ -14,18 +14,23 @@
  *  limitations under the License.
  */
 
-import smithy4s.aws._
 import cats.effect._
+import com.amazonaws.cloudwatch._
+import com.amazonaws.ec2._
+import org.http4s.client.Client
 import org.http4s.client.middleware._
 import org.http4s.ember.client.EmberClientBuilder
-
-import com.amazonaws.cloudwatch._
+import smithy4s.aws._
 
 object Main extends IOApp.Simple {
 
   type NextToken = String
 
-  override def run: IO[Unit] = cloudWatchClientResource.use(cloudWatchClient =>
+  override def run: IO[Unit] = awsEnvironmentResource.use { awsEnvironment =>
+    // Per
+    // https://disneystreaming.github.io/smithy4s/docs/protocols/aws/aws/#awsquery,
+    // CloudWatch is one of a few services that use the ec2query protocol.
```

Comment:
Oops! Nice catch, thanks.
```suggestion
    // CloudWatch is one of a few services that use the awsQuery protocol.
```